homework/calculate/calculate.hpp

```diff
@@ -1,7 +1,24 @@
 #pragma once
+
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    int result;
+    if (command == "add") {
+        result = first + second;
+    } else if (command == "subtract") {
+        result = first - second;
+    } else if (command == "multiply") {
+        result = first * second;
+    } else if (command == "divide") {
+        if (second) {
+            result = first / second;
+        } else {
+            return "Division by 0";
+        }
+    } else {
+        return "Invalid data";
+    }
+    
+    return std::to_string(result);
```

Comment:
You can also immediately return values. But this is my preference. Both solutions are good.